pkg/run/v0/aws/aws.go

```diff
@@ -81,11 +83,18 @@ func Run(plugins func() discovery.Plugins, name plugin.Name,
 	if err != nil {
 		return
 	}
+
+	var elbPlugin loadbalancer.L4
+	elbClient := elb.New(builder.Config)
+	elbPlugin, err = aws_loadbalancer.NewELBPlugin(elbClient, "elb-lb")
+	if err != nil {
```

Comment:
This here needs to be changed...

1. The ELB L4 plugin needs to bind to an actual ELB specified by name (the `"elb-lb"`) right now.   So this can't be hard-coded.
2. There's a way to pass in the name of the ELB via the `DefaultOptions`/`Options` struct but let's keep it simple right now...  let's define an environment variable... 


```
import "github.com/docker/infrakit/pkg/run/local"
...
const EnvELBName = "INFRAKIT_AWS_ELB_NAME"
...
aws_loadbalancer.NewELBPlugin(elbClient, local.Getenv(EnvELBName, "myDefaultELB"))

```

Reply:
You should be able to compile this and test...

```
make build/infrakit

INFRAKIT_AWS_ELB_NAME=myRealELB infrakit plugin start aws 
```
Then in another shell
```
infrakit -h
```
should give you a list of objects, one of which should be `aws/myRealELB`.   Then you can test by interactively add routes and backends to it...
```
infrakit aws/myRealELB routes add -h  # shows help
infrakit aws/myRealELB backends add -h # shows help
```
I'd quickly test it this way to verify it's possible to manually add and remove routes/ backends.  Once that's good, then we can hook it up to the ingress controller because we know the L4 plugin can fulfill its contract.